sale_invoice_policy/models/product_template.py

```diff
@@ -1,54 +1,18 @@
 # Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
-    default_invoice_policy = fields.Selection(
-        [("order", "Ordered quantities"), ("delivery", "Delivered quantities")],
-        string="Default Invoicing Policy",
-        help="Ordered Quantity: Invoice based on the quantity the customer "
-        "ordered.\n"
-        "Delivered Quantity: Invoiced based on the quantity the vendor "
-        "delivered (time or deliveries).",
-        default=lambda self: self.env["ir.default"].get(
-            "res.config.settings", "default_invoice_policy"
-        ),
-    )
+    def _default_invoice_policy(self):
```

Comment:
Is this really needed? Isn't this already done by default?

Reply:
By default it's a computed field. I think we can leave this (by configuration you can choose the default value)

Reply:
Sorry, we don't need it because we already have this by default https://github.com/odoo/odoo/blob/16.0/addons/sale/models/res_config_settings.py#L11